examples/all-clusters-app/all-clusters-common/src/smco-stub.cpp

```diff
@@ -15,9 +15,177 @@
  *    limitations under the License.
  */
 
-#include <lib/core/DataModelTypes.h>
+#include <app/clusters/smoke-co-alarm-server/SmokeCOTestEventTriggerDelegate.h>
+#include <app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.h>
 
-bool emberAfPluginSmokeCoAlarmSelfTestRequestCommand(chip::EndpointId endpointId)
+#include <platform/CHIPDeviceLayer.h>
+
+using namespace chip;
+using namespace chip::app::Clusters::SmokeCoAlarm;
+using namespace chip::DeviceLayer;
+
+namespace {
+
+constexpr const uint16_t kSelfTestingTimeout = 10;
```

Comment:
add a unit of measurement like `Sec` or `Ms` or similar to the constant name.

Reply:
It's changed.